perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -1215,6 +1215,27 @@ public RichMember setSponsoredMember(PerunSession session, Vo vo, User userToBeS
 		return membersManagerBl.getRichMember(session, membersManagerBl.setSponsoredMember(session, vo, userToBeSponsored, namespace, password, sponsor, true));
 	}
 
+	@Override
+	public Map<String, Map<String, String>> createSponsoredMembers(PerunSession session, Vo vo, String namespace, List<String> names, User sponsor) throws PrivilegeException {
+		Utils.checkPerunSession(session);
+		Utils.notNull(vo, "vo");
+		Utils.notNull(namespace, "namespace");
+		Utils.notNull(names, "names");
+
+		if (sponsor == null) {
+			//sponsor is the caller
```

Comment:
You should add the whole information 'sponsor is the caller, authorization is checked in Bl'.

Reply:
Added.